firefox-ios/Shared/Strings.swift

```diff
@@ -2234,6 +2234,12 @@ extension String {
                 value: "App Icon",
                 comment: "On the app icon customization screen where you can select an alternate icon for the app, this is the title displayed at the top of the screen.")
 
+            public static let ContributorCreditSubtitle = MZLocalizedString(
+                key: "Settings.AppIconSelection.AppIconNames.ContributorCredit.Subtitle.v139",
```

Comment:
I'm guessing we don't want the "AppIconNames" in the namespacing here

```suggestion
                key: "Settings.AppIconSelection.ContributorCredit.Subtitle.v139",
```

Reply:
Good catch, thank you!!